scripts/v2-migration/products/input/V2 Onchain Product Info - Products.csv

```diff
@@ -0,0 +1,85 @@
+Product Id,Product Address ,Name,Product Type,Yield Token Address,Cover Assets,Initial Price Ratio,Capacity Reduction Ratio,Is Deprecated,Use Fixed Price,Allowed Pools,IPFS Hash,IPFS data

```

Comment:
Took this from the V2 Onchain Product Info sheet.

Happy to use another source, but this was ready to go as is.